common/scala/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 
```

Comment:
All of those should be removable to complete the pull-request, modulo spray-json.